internal/lib/sandbox/builder/sandbox.go

```diff
@@ -0,0 +1,443 @@
+package builder
```

Comment:
ah this should be in `internal/lib/sandbox/builder.go` IMO (keep it internal in the package, that's the point of having the builder be next to the sandbox, so we don't need to expose setters on fields we won't set past the first instance

Reply:
I actually tried the problem is it basically creates a circular dependency. The current factory/sandbox requires factory/container. factory/container in turns requires lib/sandbox. If we move factory/sandbox this means we are creating an circular dependency.